ui/admin/app/components/form/host-catalog/static/index.hbs

```diff
@@ -71,32 +71,6 @@
         </G.RadioCard>
       {{/each}}
     </Hds::Form::RadioCard::Group>
-
-    {{#if @model.isPlugin}}
```

Comment:
This block is not being used as we [default](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/app/routes/scopes/scope/host-catalogs/new.js#L58) to `aws` if the model type is `plugin` and the [root](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/app/components/form/host-catalog/index.hbs#L7) `index.hbs` takes you to the aws form.